acceptance/support/capybara.rb

```diff
@@ -1,6 +1,6 @@
 Capybara.register_driver :selenium do |app|
   chrome_options = Selenium::WebDriver::Chrome::Options.new.tap do |o|
-    o.add_argument '--headless'
+    # o.add_argument '--headless'
```

Comment:
```suggestion
    o.add_argument '--headless'
```

Reply:
You're too quick! I'd have realised as soon as I looked at circle, as all the tests would have failed!